prow/config/external_plugins_config.yaml

```diff
@@ -455,8 +450,8 @@ ti-community-label:
       - 'special-week'
       - 'v1.0'
       - 'v1.1'
-      - 'v1.2'
-      - 'version-specific-changes-required'
+      - 'for-future-release'
```

Comment:
v1.2 should be retained. We'll have a v1.2-beta release next week.

Reply:
```suggestion
      - 'v1.2'
      - 'for-future-release'
```